local-dev/compile-proto.sh

```diff
@@ -84,4 +86,13 @@ $docker_run protoc \
 echo "Fixing imports for protobuff JS for more information see the README file"
 find "${WORK_DIR}"/dashboard/apps/web/littlehorse-public-api/ -type f -readable -writable -exec sed -i "s/import _m0/import * as _m0/g" {} \;
 
+
+# This segment is for the sdk-js should replace the dashboard version eventually
+$docker_run protoc \
+	--plugin=/usr/local/lib/node_modules/ts-proto/protoc-gen-ts_proto \
```

Comment:
Is it possible to combine this with the other one on line 44?

Reply:
Sadly no, since they generate different outputs